### PR TITLE
Fixed the filter and sort bug and update a few lines of documentation

### PR DESCRIPTION
**Pull request by RyanQiu1**

### CODE REVIEW

File: src/main/java/seedu/techtrack/model/role/Website.java [hunk 1]
```diff
@@ -11,7 +11,7 @@ public class Website {
 
     public static final String MESSAGE_CONSTRAINTS = "Websites should be of the format www.domain.com"
             + " and adhere to the following constraints:\n"
-            + "1. The first part should only contain www.\n"
+            + "1. The first part should only contain 'www'.\n"
```
Comment (dsja612) [inline src/main/java/seedu/techtrack/model/role/Website.java:14]: Good! This falls within the restrictions of v1.4

File: src/main/java/seedu/techtrack/model/ModelManager.java [hunk 2]
```diff
@@ -133,7 +133,8 @@ public void updateFilteredRoleList(Predicate<Role> predicate) {
 
     @Override
     public void displaySortedSalaryList(OrderParser orderParser) {
-        List<Role> roles = filteredRoles.sorted((r1, r2) -> {
+        FilteredList<Role> filteredRoleCopy = new FilteredList<>(this.roleBook.getRoleList());
+        List<Role> roles = filteredRoleCopy.sorted((r1, r2) -> {
```
Comment (dsja612) [inline src/main/java/seedu/techtrack/model/ModelManager.java:137]: Clean functional code

**Review by dsja612 — APPROVED** [review]
LGTM

**Review by amoonguss1 — APPROVED** [review]
LGTM! Good job on fixing the filter and sort bug!

**Pull request merged**